services/datamanager/builtin/builtin.go

```diff
@@ -147,6 +150,8 @@ type builtIn struct {
 	syncerConstructor   datasync.ManagerConstructor
 	filesToSync         chan string
 	maxSyncThreads      int
+	syncConfigUpdated   bool
+	reinitSyncer        bool
```

Comment:
nit, can this be renamed? I'm not sure what this represents as-is. something like, "syncer needs to be re-initialized"?

Reply:
done